browser-test/src/admin/admin_application_statuses.test.ts

```diff
@@ -344,6 +344,14 @@ test.describe('view program statuses', () => {
 
       expect(await adminPrograms.getNoteContent()).toBe(noteText)
     })
+    test('allow notes to be exported', async ({adminPrograms}) => {
+      await adminPrograms.editNote('Note is exported')
+      await adminPrograms.expectNoteUpdatedToast()
+      const noApplyFilters = false
+
+      const csvContent = await adminPrograms.getCsv(noApplyFilters)
+      expect(csvContent).toContain('Note is exported')
+    })
 
```

Comment:
**suggestion:** I think it's worth adding a test here too "note gets overwritten" or something like that, which sets the note again and we see the new note instead of the original one.

Reply:
sure. moved it to a new test

Reply:
I guess I meant to confirm that it overwrites the old note. So maybe adding and expectation that the csv does not contain 'Some note content', which was the original note

Reply:
oh, in the browser tests? I have added it in the unit tests. no problem adding here too.